Makefile.in

```diff
@@ -415,22 +415,21 @@ testobsoletes:
             `date -u +dev/log/testobsoletesN1_%Y-%m-%d-%H-%M`; rm testobsoletesN1.in )
 	( rm wsp.g )
 
-cygwin:	extern
-	cnf/mkversionheader.sh bin/$(GAPARCH)/gap_version.h
-	( cd bin/$(GAPARCH) ; $(MAKE) gapdll ABI='$(ABI)' CC='$(CC)' GMP_VER='$(GMP_VER)' GMP_CFLAGS='$(GMP_CFLAGS)' GMP_LIBS='$(GMP_LIBS)' CONFIGNAME='$(CONFIGNAME)' )
-	chmod +x bin/gap-$(CONFIGNAME).sh
+ifeq ($(shell uname -o), Cygwin)
+default: cygwin
+endif
+
+cygwin: compile
 	chmod +x bin/*.bat
-	cp bin/$(GAPARCH)/gapw95.exe bin
-	cp bin/$(GAPARCH)/gap.dll bin
+	chmod +x cnf/instcygwinterminfo.sh
+	cnf/instcygwinterminfo.sh
+	( if which peflags ; then peflags --cygwin-heap=2048 bin/$(GAPARCH)/gap.exe ; fi )
 	cp /bin/cygncurses-10.dll /bin/cygncursesw-10.dll \
 	   /bin/cygpanel-10.dll /bin/cygpanelw-10.dll /bin/cygwin1.dll \
 	   /bin/libW11.dll /usr/bin/rxvt.exe /usr/bin/regtool.exe \
 	   /bin/cygreadline7.dll /bin/cyggcc_s-1.dll /bin/cygpopt-0.dll \
-	   /bin/cygstart.exe /bin/mintty.exe bin
-	chmod +x cnf/instcygwinterminfo.sh
-	cnf/instcygwinterminfo.sh
-	( if which peflags ; then peflags --cygwin-heap=2048 bin/gapw95.exe ; fi )
-	
+	   /bin/cygstart.exe /bin/mintty.exe bin/$(GAPARCH) || :
+
 winbinp:
```

Comment:
what is `winbinp` by the way? IIRC, this was some experiment by Max N. and may be removed now.